webplugin/js/app/kommunicate.js

```diff
@@ -116,6 +116,45 @@ $applozic.extend(true, Kommunicate, {
             }, SET_TIMEOUT_DURATION);
         }
     },
+    updateConversationMetadata: function (conversationMetadata) {
+        if (
+            conversationMetadata != null &&
+            typeof conversationMetadata !== 'undefined'
+        ) {
+            var _metadata = Object.assign({},conversationMetadata);
```

Comment:
What's the reason for using _ ?

Reply:
we already have some declaration named `metadata`.

Reply:
where is it?

Reply:
![image](https://user-images.githubusercontent.com/22856546/110589386-5a9c3500-819c-11eb-9b85-89fc1e21a776.png)
-
![image](https://user-images.githubusercontent.com/22856546/110589484-7dc6e480-819c-11eb-9122-e2d104ba01d4.png)

Reply:
with that logic, then above functions should also not work because it is already present in 1st function, then how come it is working on 2nd function as well?